__tests__/app.test.js

```diff
@@ -171,3 +171,37 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe.only("GET /api/articles", () => {
+  test("200: responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("404: responds with not found if given wrong path", () => {
+    return request(app)
+      .get("/api/barnacles")
+      .expect(404)
+      .then(({ body: { msg } }) => {
```

Comment:
Try to be consistent with destructuring. For example your 200 test has the destructuring as below

```js
.then(({body}) => {
  const {article} = body
})
```

whereas the 404 test shows

```js
 .then(({ body: { msg } }) => {}
```